Reason: Skip time-sensitive test which might fail on busy machines
Upstream: No

--- a/resources/images/images_golden_integration_test.go
+++ b/resources/images/images_golden_integration_test.go
@@ -302,6 +302,7 @@
 }

 func TestImagesGoldenProcessMisc(t *testing.T) {
+	t.Skip("skip time-sensitive test")
 	t.Parallel()

 	if imagetesting.SkipGoldenTests {
@@ -347,6 +348,7 @@
 }

 func TestImagesGoldenProcessWebP(t *testing.T) {
+	t.Skip("skip time-sensitive test")
 	t.Parallel()

 	if imagetesting.SkipGoldenTests {
@@ -421,6 +423,7 @@
 }

 func TestImagesGoldenWebPAnimation(t *testing.T) {
+	t.Skip("skip time-sensitive test")
 	t.Parallel()

 	if imagetesting.SkipGoldenTests {
@@ -527,6 +530,7 @@
 }

 func TestImagesGoldenConfigLossyVsQuality(t *testing.T) {
+	t.Skip("skip time-sensitive test")
 	t.Parallel()

 	if imagetesting.SkipGoldenTests {
